_src/css/_body.scss

```diff
@@ -17,96 +17,50 @@
     box-shadow: 3px 2px 10px 0 white;
     margin: 25px;
     display: flex;
-    flex-direction: column;
+    justify-content: space-around;
     border-radius: 2%;
   }
 
-  &__info{
-    display:flex;
-    height: 90%;
-  }
-
   &__frame {
     width: 30%;
     display: flex;
     flex-direction: column;
     position: relative;
     justify-content: center;
     align-items: center;
+    bottom: 15px;
   }
 
-  &__frame::after{
-    content: "";
-    position: absolute;
-    background-color:gray;
-    width: 1px;
-    height: 50px;
-    left: 160px;
-    top: 100px;
-  }
   
   &__image{
     background-color: blanchedalmond;
-    border: grey 2px solid;
+    border: lightgrey 3px solid;
     border-radius: 50%;
-    height: 140px;
-    width: 140px;
+    height: 120px;
+    width: 120px;
   }
 
   &__content {
-    width: 70%;
+    width: 60%;
     display: flex;
     flex-direction: column;
   }
 
   &__title{
+    font-size: 25px;
     display: flex;
-    justify-content: center;
+    justify-content: flex-start;
   }
 
   &__text{
+    margin-top: 35px;
```

Comment:
There's no need for the margin to be this much. We can use the default marging of this element to set the correct spacing between the title and text.